datasets/commoncrawl.yaml

```diff
@@ -1,53 +1,67 @@
 Name: Common Crawl
-Description: A corpus of web crawl data composed of over 25 billion web pages.
-Documentation: http://commoncrawl.org/the-data/get-started/
-Contact: http://commoncrawl.org/connect/contact-us/
-ManagedBy: "[Common Crawl](http://commoncrawl.org/)"
+Description: A corpus of web crawl data composed of over 50 billion web pages.
+Documentation: https://commoncrawl.org/the-data/get-started/
+Contact: https://commoncrawl.org/connect/contact-us/
+ManagedBy: "[Common Crawl](https://commoncrawl.org/)"
 UpdateFrequency: Monthly
 Tags:
   - aws-pds
   - encyclopedic
   - machine learning
   - natural language processing
   - internet
-License: This data is available for anyone to use under the [Common Crawl Terms of Use](http://commoncrawl.org/terms-of-use/)
+License: This data is available for anyone to use under the [Common Crawl Terms of Use](https://commoncrawl.org/terms-of-use/)
 Resources:
   - Description: Crawl data (WARC and ARC format)
     ARN: arn:aws:s3:::commoncrawl
     Region: us-east-1
     Type: S3 Bucket
 DataAtWork:
   Tutorials:
-    - Title: Large-scale graph mining with Spark
```

Comment:
Is this tutorial no longer relevant? I'm somewhat hesitant to remove a tutorial, even if replaced by others

Reply:
It's a good if not excellent tutorial about Spark graph mining. And, yes, it uses Common Crawl data. Unfortunately, there is no code (even no snippets) shared how to extract links from the WARC files and span up the graph. This isn't trivial, at least, not for beginners starting to use the data.